app.rb

```diff
@@ -0,0 +1,136 @@
+require_relative './person'
+require_relative './rental'
+require_relative './student'
+require_relative './teacher'
+require_relative './book'
+require_relative './classroom'
+
+class App
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  # Code to list all books
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/224444416-ebd0b390-3d85-428d-8146-e5d5dd3a0d7c.png)

- [ ] It would be nice if you add `ruby main.rb` command in the section shown below to direct users of your application, to how they can use your application 👍